src/renderer/tests/basic.rs

```diff
@@ -570,6 +570,82 @@ fn ok_many_variable_blocks() {
     assert_eq!(render_template(&tpl, &context).unwrap(), expected);
 }
 
+#[test]
+fn can_expand_block_multiple_times_with_useblock() {
+    let result = render_template(
+        r#"
+{%- block test -%}Hello World{%- endblock test -%}
+{%- useblock test %} {% useblock test -%}
+        "#,
+        &Context::new()
+    );
+
+    assert_eq!(result.unwrap(), "Hello World Hello World");
+}
+
+#[test]
+fn can_expand_block_multiple_times_with_useblock_nested() {
+    let result = render_template(
+        r#"
+{%- block test -%}Hello World{%- endblock test -%}
+{%- block test2 -%}{%- useblock test %} {% useblock test -%}{%- endblock test2 -%}
+{%- useblock test2 -%}
+        "#,
+        &Context::new()
+    );
+
+    assert_eq!(result.unwrap(), "Hello World Hello World");
+}
+
+#[test]
+fn can_expand_block_with_useblock_with_inheritance() {
+    let mut tera = Tera::default();
+    tera.add_raw_template("base", "{% useblock test %}").unwrap();
+    tera.add_raw_template(
+        "index",
+        "{% extends \"base\" %}{% block test %}Hello World{% endblock test %}"
```

Comment:
that seems a bit wrong to me, this is not re-using the block test it's declaring it first in the base template via useblock and then overriding it in the child template? This shouldn't work

Reply:
I think the intention here (unless I'm missing something) is to have the content of the block defined by the upper-level template, and referenced in the base.

The example from #265 was a title block (with content provided from the "index" template), which gets used within both a `<title>` tag and a `<meta>` tag in the base template.